Symfony/CS/Error/Internal.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Error;
+
+class Internal extends AbstractError
```

Comment:
Why do we need that tree? Abstract, Internal or External. Are they going to be different?